modules/programs/i3status-rust.nix

```diff
@@ -0,0 +1,179 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.i3status-rust;
+
+  # This is in nixpkgs now, but not in 20.03
+  # TODO: Take from upstrem (https://github.com/NixOS/nixpkgs/blob/master/pkgs/pkgs-lib/formats.nix) once HM no longer tracks 20.03
```

Comment:
The master branch of Home Manager follows the unstable Nixpkgs channel, not 20.03. So you can remove this.

Reply:
Good point, so my comment wrong and I've corrected it in c414b30667b34289431ee43e51c267645f799d2f.

I'm currently using HM as a Nixos module on 20.03, which means the based `pkgs` come from 20.03 still. For that reason selfishly, I'd like to keep this shim around for a few months until 20.09 is out.

Unless you have a better idea?

Reply:
@rycee, with 20.09 available as a channel, this fork is now removed and taken from upstream!